app/src/main/kotlin/com/intive/picover/parties/viewmodel/PartiesViewModel.kt

```diff
@@ -3,23 +3,37 @@ package com.intive.picover.parties.viewmodel
 import androidx.compose.runtime.MutableState
 import androidx.compose.runtime.mutableStateOf
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.intive.picover.parties.model.toUI
 import com.intive.picover.parties.repository.MockedPartiesRepository
 import com.intive.picover.parties.state.PartiesState
+import com.intive.picover.parties.state.PartiesState.Error
+import com.intive.picover.parties.state.PartiesState.Loaded
+import com.intive.picover.parties.state.PartiesState.Loading
 import dagger.hilt.android.lifecycle.HiltViewModel
 import javax.inject.Inject
+import kotlinx.coroutines.launch
 
 @HiltViewModel
 class PartiesViewModel @Inject constructor(
 	private val partiesRepository: MockedPartiesRepository,
 ) : ViewModel() {
 
-	val parties: MutableState<PartiesState> = mutableStateOf(PartiesState.Loading)
+	val parties: MutableState<PartiesState> = mutableStateOf(Loading)
 
 	init {
-		loadParties()
+		viewModelScope.launch {
```

Comment:
I am not sure `launch` from `viewModelScope` should be here maybe we can use it in `loadParites()`

Reply:
for sure not, especially when we want to use a retry mechanism soon :D i wanted to fix it in separate task for error/retry mechanism impl but i will fix it now :)